docs/.vuepress/components/Home/Contest2.vue

```diff
@@ -5,7 +5,8 @@
         <span> Mapmaking Contest 2 </span> <br> Old is New
       </h1>
       <div class="guide-part-body">
-        <p> From November 2020 until just after the new year, we had our second mapmaking contest. $550 in prizes went to <a href=https://imgur.com/gallery/JVvUSQT>nine amazing maps</a> as judged by a panel and by the Rocket League Community. Stay tuned for more in the future!
```

Comment:
I overhauled the contest portion anyway with my unmerged changes, but thanks for cleaning this up.